app/workers/ProcessQueue/__init__.py

```diff
@@ -152,13 +151,13 @@ def select_concepts_to_post(
     for new_content_detail in new_content_details:
         try:
             # fields
-            if content_type == 15:
+            if content_type == "scanreportfield":
```

Comment:
For these hardcoded strings that you're reusing all over the place, can you create a constants file, like `constants.py` in the appropriate place to store these. Then import them and you only need to change in one place